admin-core/assets/src/dashboard-app/pages/settings/block-settings/InstagramUsers.js

```diff
@@ -331,9 +331,11 @@ const InstagramUsers = () => {
 	const generateDP = ( user ) => {
 		switch ( user.userType.toLowerCase() ) {
 			case 'personal':
+				// Will be the profile picture.
+				const profileUserName = 'string' === typeof user.userName ? user.userName[0].toUpperCase() : UAGB_Block_Icons['instagram-feed'];
```

Comment:
```suggestion
				const profileUserName = 'string' === typeof user.userName ? user?.userName[0].toUpperCase() : UAGB_Block_Icons['instagram-feed'];
```

Can there be nay chance of not getting userName[0]

@bsf-zanev @AbhijitNage123

Reply:
@adi3890 Might be possible. We can use `?` mark to check No issue. Updated code as suggested.

Reply:
@AbhijitNage123 - We can do this to avoid any unwanted behaviour.

https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3406/files#r1292998985

Reply:
Resolved